### PR TITLE
populate Getting Started for pkg dev workshop

### PR DESCRIPTION
**Pull request by lindsaycarr**
completing #159 & fixing in-line links

### CODE REVIEW

File: content/r-package-dev/Getting_Started.Rmd [hunk 1]
```diff
@@ -29,12 +29,49 @@ opts_chunk$set(
...
+Software installation:
+
+- [R](https://cran.rstudio.com/bin/windows/base/) (latest version)
+- [RStudio](https://www.rstudio.com/products/rstudio/download2/) (>1.0)
+- [RTools](https://cran.r-project.org/bin/windows/Rtools/) (most recent frozen version)
+- R packages: devtools, roxygen2, testthat, knitr
+
+## Suggested prerequisite knowledge
+
+This course discusses advanced topics in programming. Transitioning from scripting mentality to package development can be challenging; we recommend you have advanced knowledge of R programming to use this curriculum. Please refer to the list below to see if you qualify. In addition, putting time and effort into package development is more useful if you have an existing script that could be useful to yourself and others if it were turned into a package.
```
Comment (wdwatkins) [inline content/r-package-dev/Getting_Started.Rmd]: missed an `a` in the second sentence

File: content/r-package-dev/Getting_Started.Rmd [hunk 2]
```diff
@@ -29,12 +29,49 @@ opts_chunk$set(
...
+
+- loading files into data.frames.
+- differentiating data structures and data types.
+- indexing vectors, data.frames, or lists.
+- creating scatter, line, or boxplots and saving the output as a PNG/JPG.
+- writing for loops.
+- using logical statements (>, >=, ==).
+- writing conditional statements (if, if-else).
+- installing, loading, and using additional packages.
+- troubleshooting/decrypting error messages.
```
Comment (wdwatkins) [inline content/r-package-dev/Getting_Started.Rmd]: Probably the biggest thing there would be writing functions right?

Comment (lindsaycarr) [inline content/r-package-dev/Getting_Started.Rmd]: WAHHH can't believe we forgot that!

**Comment by lindsaycarr** [comment]
Fixed based on @wdwatkins feedback. Merging now.

**Pull request merged**